docs/contributors_guide/maintenance/iso_release_procedures/Pre_Release_Checklist.rst

```diff
@@ -73,22 +73,23 @@ the RPMs for those projects exist by executing the ``pkg:check_published`` Rake
 
       If you see a message like
       ``Warning:  Unable to generate build-specific YUM cache``, your
-      results are invalid, as connection to `PackageCloud`_ failed.
+      results are invalid, as connection to the `SIMP Download Server`_ failed.
 
-#. Manually verify the appropriate ``simp-doc`` RPM exists at `PackageCloud`_.
+#. Manually verify the appropriate ``simp-doc`` RPM exists on the
+`SIMP Download Server`_.
```

Comment:
Similar comments to the above; if it's just going link outside of the documentation to a top-level FQDN, use the URL for clarity.